core/src/main/java/com/datastax/oss/driver/internal/core/util/concurrent/CompletableFutures.java

```diff
@@ -72,6 +73,40 @@
       List<CompletionStage<T>> inputs, Runnable callback, Executor executor) {
     allDone(inputs).thenRunAsync(callback, executor).exceptionally(UncaughtExceptions::log);
   }
+  /**
+   * @return a completion stage that completes when all inputs are successful, or fails if any of
+   *     them failed.
+   */
+  public static <T> CompletionStage<Void> allSuccessful(List<CompletionStage<T>> inputs) {
```

Comment:
Do we use some different implementation of `CompletionStage` than `CompletableFuture`?
If no why don't just use `CompletableFuture.allOf()`?

Reply:
Good catch, it didn't occur to me to check if the JDK had a built-in method for this version.

However, `CompletableFuture.allOf`:
* deals with `CompletableFuture` instances, not `CompletionStage`, so we would have to convert each one with `toCompletableFuture()`
* only takes a vararg parameter. In our generated code it's a list, so we'd have to add a `toArray` call.
* only reports the first failure that it encounters. I think it's nice to have them all at once, so that if there are multiple errors you have an opportunity to fix them all at once.